apps/main/utilities.py

```diff
@@ -1,38 +1,42 @@
-import datetime
+from datetime import date, timedelta
 
 from apps.afisha.models import Event
 from apps.articles.models import BlogItem, NewsItem
 from apps.core.models import Settings
 from apps.info.models import Festival, Place
-from apps.library.models import ProgramType
+from apps.library.models import Play, ProgramType
 from apps.main.models import Banner
 
 
 class MainObject:
     def add_first_screen_data(self):
         main_add_first_screen = Settings.get_setting("main_add_first_screen")
         if main_add_first_screen:
-            self.first_screen_title = Settings.get_setting(
-                "main_first_screen_title"
-            )
-            self.first_screen_url_title = Settings.get_setting(
-                "main_first_screen_url_title"
-            )
-            self.first_screen_url = Settings.get_setting(
-                "main_first_screen_url"
-            )
+            title = Settings.get_setting("main_first_screen_title")
+            url_title = Settings.get_setting("main_first_screen_url_title")
+            url = Settings.get_setting("main_first_screen_url")
+            self.first_screen = dict()
+            self.first_screen["title"] = title
+            self.first_screen["url_title"] = url_title
+            self.first_screen["url"] = url
```

Comment:
Можно эти две строчки так схлопнуть. 
```suggestion
            self.first_screen ={"title": title, "url_title": url_title,"url": url}
```

И все остальные тоже бы так сократить

Reply:
Согласен, пожалуй так лучше.